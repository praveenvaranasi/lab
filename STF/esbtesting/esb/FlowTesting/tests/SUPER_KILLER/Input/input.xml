<Counter> 1 </Counter>
