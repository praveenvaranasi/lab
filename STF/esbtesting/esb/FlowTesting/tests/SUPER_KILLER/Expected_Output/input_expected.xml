<Result>
    <Code>0</Code>
    <Message>
        <Current>2</Current>
        <Last>-1</Last>
    </Message>
</Result>
